frontend/views/pages/GroupChat.vue

```diff
@@ -117,6 +117,7 @@ import ChatMembers from '@containers/chatroom/ChatMembers.vue'
 import { OPEN_MODAL } from '@utils/events.js'
 import { MenuParent, MenuTrigger, MenuContent, MenuItem, MenuHeader } from '@components/menu/index.js'
 import { CHATROOM_PRIVACY_LEVEL } from '@model/contracts/shared/constants.js'
+import '../containers/chatroom/chat-mentions/RenderMessageWithMarkdown.js'
```

Comment:
Is this importing necessary?

Reply:
Yes. as far as I know the Vue render function cannot be exported as SFC, so the registration of the component (executing `Vue.component(...)`) has to be executed somewhere in the app. This could've been done anywhere (e.g. `main.js`) but this component is only relevant in the chat area of the app. So I exported the file here instead.

Reply:
I actually change `RenderMessageWithMarkdown` as an exportable object and have registered it as a child component of `MessageBase.vue` in the latest update.